### ISSUE TITLE
Thermoworks is releasing a new gateway for RFX, have you had a chance to analyze it yet?

### ISSUE THREAD

**Issue by cribbstechnologies** [G-Two/smoke-x-receiver#21]
I just found your repo and I'm about to purchase the hardware to interface my Smoke X4 with HomeAssistant using your library. Thermoworks released a different platform called RFX, have you had a chance to look at that yet? It looks like the probes connect to a gateway via RF and then that connects back to thermoworks HQ via wifi and I wonder if something along the lines of this software could sniff out the data from the probes somehow.

**Comment by G-Two** [comment]
Interesting product! I haven't had a chance to buy the RFX yet (or analyze its signals). But it should definitely be possible to to develop a similar library/application to directly collect the data from the RF signal once the protocol is analyzed.

**Comment by neillbell** [comment]
I see from the Thermoworks site that the RFX devices are 433MHz, so they are not LoRa.

Someone has already [mentioned it](https://github.com/merbanan/rtl_433/issues/3041) on the rtl_433 site. If these devices can be decoded with rtl_433 software, then it should be very easy to listen to access these with a SDR. Even interfacing to Home Assistant should be simple since rtl_433 can be configured to send its output to MQTT.

**Comment by jazzmonger** [comment]
thermoworks has an api to their cloud data.  
https://pypi.org/project/thermoworks-cloud/
wouldn't this be a way to get data into HA easily?  I know its not local, but who cares about beef temperatures being in the cloud?

**Comment by G-Two** [comment]
@jazzmonger You're correct for the RFX product. It is already sending data to the cloud, so using the API is absolutely the easiest way to get the data into Home Assistant. 

I don't think folks are bothered by their beef temperatures being in the cloud from a privacy or security standpoint. Many are just concerned if a product *is required* to communicate to the cloud since that could impede the product's long-term usability if the vendor ever decides to stop supporting it.

**Comment by cribbstechnologies** [comment]
I got the RFX and recorded the signals with a 433 radio but I've been
unsuccessful in decoding them.

Another BBQ temp company, I don't remember which, was offline recently
during a holiday weekend rendering their hardware useless. I'd like to
avoid that at all costs.

**Comment by jazzmonger** [comment]
>Another BBQ temp company, I don't remember which, was offline recently
during a holiday weekend rendering their hardware useless. I'd like to
avoid that at all costs.

That's another reason I went with the RFX - The collection gateway is **local** and the app communicates locally with the gateway - You don't need internet to use it.  But, the gw does sent that info on to the cloud if it is available. 

Kinda the best of both worlds. 

Anyway, I'd really like to see RFX data in HA... I'm ready....

![](https://community-assets.home-assistant.io/original/4X/9/9/7/997880ef35622818ca7e28ec5be002e13278fdf6.png)
